unity/UnityEmbedHost.Generator/CoreCLRHostGenerator.cs

```diff
@@ -73,7 +73,22 @@ static unsafe partial class CoreCLRHost
         {
             sb.AppendLine("    [System.Runtime.InteropServices.UnmanagedCallersOnly]");
             string signature = methodSymbol.FormatMethodParametersForMethodSignature();
-            sb.AppendLine($"    static {methodSymbol.ReturnType} {methodSymbol.Name}_native({signature}) => {methodSymbol.Name}({FormatMethodParametersNames(methodSymbol)});");
+            sb.AppendLine($"    static {methodSymbol.ReturnType} {methodSymbol.Name}_native({signature})");
+            sb.AppendLine("    {");
+            sb.AppendLine("        try");
+            sb.AppendLine("        {");
+            sb.AppendLine($"            return {methodSymbol.Name}({FormatMethodParametersNames(methodSymbol)});");
+            sb.AppendLine("        }");
+            sb.AppendLine("        catch (System.Exception e)");
+            sb.AppendLine("        {");
+            sb.AppendLine("            var bytes = System.Text.Encoding.UTF8.GetBytes(e.ToString());");
+            sb.AppendLine("            fixed (byte* p = bytes)");
+            sb.AppendLine("            {");
+            sb.AppendLine("                 CoreCLRHostNative.unity_log(p);");
+            sb.AppendLine("            }");
+            sb.AppendLine("            return default;");
```

Comment:
I forgot a note I had.  Someone mentioned calling `Abort`.  Should I do that instead of returning?

Reply:
I am up in the air on this. I *think* aborting, at least in this stage of development, would be good? I don't know any case where an exception should be ignored ATM.

Reply:
I would lean that way as well.